drivers/ethernet/eth_smsc911x.c

```diff
@@ -578,12 +578,13 @@ static struct net_pkt *smsc_recv_pkt(struct dehive *dev, u32_t pkt_size)
 
 	/* Discard FCS */
 	{
-		u32_t dummy = SMSC9220->RX_DATA_PORT;
+		u32_t __unused dummy = SMSC9220->RX_DATA_PORT;
```

Comment:
Can't we just remove the dummy block here?

Reply:
No, this line does flush the rx data port register by reading it. Original code needed it, so I guess this is still valid.

Reply:
Ah, ok then.